2-mandatory/8-password-validator.js

```diff
@@ -22,8 +22,20 @@ Expected Result:
 PasswordValidationResult=  [false, false, false, false, true]
 
 */
+validatePasswords(["StUFf27%", "Pl3nty!", "Jai33", "shajsaUA**&&", "Pl3nty!"]);
+const validatePasswords = (passwordArray) => passwordArray.map(isValidPassword);
```

Comment:
You are getting one error here as you are trying to refer to the variable `validatePasswords` before it has been defined. To fix this you have to place `const validatePasswords =` before you can use `validatePasswords`. So all you have to do is swap lines 25 and 26, and it will all work :)

I do this **all** the time and is one of the most common errors 😅